sentry-android-timber/src/test/java/io/sentry/android/timber/SentryTimberTreeTest.kt

```diff
@@ -215,6 +215,18 @@ class SentryTimberTreeTest {
         )
     }
 
+    @Test
+    fun `Tree adds a breadcrumb with formatted message and arguments, when provided`() {
+        val sut = fixture.getSut()
+        sut.e("test count: %d", 32)
```

Comment:
do we want another test case with %s that causing problems in https://github.com/getsentry/sentry-java/issues/1956 or do we trust that if %d works, %s will also work?

Reply:
No need, Timber does the string format, we only care about using the right property when adding the breadcrumb.